components/library/ptlib/patches/ptlib.pc.in.patch

```diff
@@ -0,0 +1,11 @@
+--- ptlib-2.10.11/ptlib.pc.in.orig	2016-07-22 20:44:31.496630089 +0200
```

Comment:
Is it really needed to add -R${libdir} when usr/lib/libpt.so.$(COMPONENT_VERSION) is delivered?